extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/ESH-INF/thing/channels.xml

```diff
@@ -120,9 +120,9 @@
 				<description>An MQTT topic that this thing will subscribe to, to receive the state. This can be left empty, the thing will be state-less then.</description>
 				<default></default>
 			</parameter>
-			<parameter name="commandTopic" type="text">
+			<parameter name="commandTopic" type="text" required="true">
```

Comment:
It machtes how it is required in the code and described in the Readme.
If the channel only has a state but is otherwise read-only, it is a `Contact`.

I thing I have done it this way, because the "stateTopic" is also optional (for a stateless, push-button like trigger switch) and an evil user could leave out "stateTopic" and "commandTopic".